### PR TITLE
Schema

### PR DESCRIPTION
**Pull request by ChunhuiY**

### CODE REVIEW

**Comment by ChunhuiY** [comment]
@LeonQC pls review when you have time

File: Schema.png
Comment (LeonQC) [inline Schema.png:1]: Why Project connect to Task?

Comment (ChunhuiY) [reply Schema.png:1]: Got it, I had modified it. Anything else?

Comment (LeonQC) [reply Schema.png:1]: Screenshot 2024-05-12 at 11 11 52 PM

Still the same, what  changed?

Comment (ChunhuiY) [reply Schema.png:1]: I think the new one can be saw in the file now. And the two demo is in the final view, I will upload them today.

**Review by LeonQC — APPROVED** [review]

**Pull request merged**